app/api/service.py

```diff
@@ -0,0 +1,26 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList
+
+from app.api.bootstrap import api
+from app.api.schema.services import ServiceSchema
+from app.models import db
+from app.models.service import Service
+
+
+class ServiceList(ResourceList):
+    """
+    List services
```

Comment:
Maybe a more verbose description would be better, like what the function does or why it does it.